CMakeLists.txt

```diff
@@ -47,10 +47,10 @@ catkin_package(
 
 include_directories(include
   ${catkin_INCLUDE_DIRS}
-  lib/InertialSenseSDK/src
+  ../../../cpp/SDK/src #Reference current version of SDK instead of git submodule
```

Comment:
We don't want to push this to the web, we need to think about how to make this easier.

Reply:
Any ideas on other options? The problem is that if the SDK gets updated to implement a feature in our ROS code, we need to be able to use the dev SDK. If it all lives on a dev branch it seems OK to me. Maybe we need to move the ros code into IS-src and then when we do a release we push an update to the external ros repo, sort of like we do with the SDK...